nilearn/datasets/atlas.py

```diff
@@ -773,3 +772,117 @@ def fetch_coords_dosenbach_2010():
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_juelich_historical(atlas_name, data_dir=None,
+                                   symmetric_split=False,
+                                   resume=True, verbose=1):
+    """Loads Juelich historical atlases from FSL if installed.
+
+    Parameters
+    ----------
+    atlas_name: string
+        Name of atlas to load. Can be:
+        cort-maxprob-thr0-1mm,  cort-maxprob-thr0-2mm,
+        cort-maxprob-thr25-1mm, cort-maxprob-thr25-2mm,
+        cort-maxprob-thr50-1mm, cort-maxprob-thr50-2mm,
+        sub-maxprob-thr0-1mm,  sub-maxprob-thr0-2mm,
+        sub-maxprob-thr25-1mm, sub-maxprob-thr25-2mm,
+        sub-maxprob-thr50-1mm, sub-maxprob-thr50-2mm,
+        cort-prob-1mm, cort-prob-2mm,
+        sub-prob-1mm, sub-prob-2mm
+
+    data_dir: string, optional
+        Path of data directory. It can be FSL installation directory
+        (which is dependent on your installation).
+
+    symmetric_split: bool, optional
+        If True, split every symmetric region in left and right parts.
+        Effectively doubles the number of regions. Default: False.
+        Not implemented for probabilistic atlas (*-prob-* atlases)
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+
+        - "maps": nibabel.Nifti1Image, 4D maps if a probabilistic atlas is
+          requested and 3D labels if a maximum probabilistic atlas was
+          requested.
+
+        - "labels": string list, labels of the regions in the atlas.
+    """
+    atlas_items = ('maxprob-thr0-1mm', 'maxprob-thr0-2mm', 'prob-1mm',
+                   'prob-2mm', 'maxprob-thr25-1mm', 'maxprob-thr25-2mm',
+                   'maxprob-thr50-1mm', 'maxprob-thr50-2mm')
+
+    if atlas_name not in atlas_items:
+        raise ValueError("Invalid atlas name: {0}. Please chose an atlas "
+                         "among:\n{1}".format(atlas_name,
+                                              '\n'.join(atlas_items)))
+
+    # For practical reasons, we mimic the FSL data directory here.
+    dataset_name = 'fsl'
+    # Environment variables
+    default_paths = []
+    for env_var in ['FSL_DIR', 'FSLDIR']:
+        path = os.getenv(env_var)
+        if path is not None:
+            default_paths.extend(path.split(':'))
+    if len(default_paths) == 0:
+        raise ValueError("Please export FSL_DIR or FSLDIR in your environ."
+                         " This can be done by sourcing the fsl.sh "
+                         "configuration file, for example.")
```

Comment:
I think that this is the same problem as Harvard Oxford. It is only shipped with FSL and we have to either to download the whole bundle of FSL data or host it ourselves on NITRC.